app/assets/javascripts/application.js

```diff
@@ -0,0 +1,3 @@
+//= require underscore
```

Comment:
Need to include this file?

Reply:
yep, see https://apidock.com/rails/String/underscore

Reply:
Yeah I am not sure if we need this either